datasetinsights/io/config_handler.py

```diff
@@ -0,0 +1,87 @@
+import logging
+import re
+import tempfile
+from pathlib import Path
+
+from yacs.config import CfgNode
+
+import datasetinsights.constants as const
+from datasetinsights.io import create_downloader
+from datasetinsights.io.exceptions import InvalidOverrideKey
+
+logger = logging.getLogger(__name__)
+_EQUAL_STR = "="
+_REGEX_PTRN = r".*?=.*"
+
+
+def prepare_config(path=None, override=None):
+    """load and override config from local or remote locations .
+
+    Args:
+        path: config location
+        override: key-value pairs to override config
+    Returns:
+        config object of type yacs.config.CfgNode
+    """
+    config = load_config(path)
+    logger.info(f"config loading from {path} completed")
+    if override:
+        logger.info(f"overriding config params= {override}")
+        override_config(override=override, config=config)
+    return config
+
+
+def load_config(path):
+    """ load config from local or remote locations .
+    Examples:
+        >>> path="gs://thea-dev/../config.yaml"
+        >>> path="https://thea-dev/../config.yaml"
+        >>> path="http://thea-dev/../config.yaml"
+        >>> path="file:///root/../config.yaml" # absolute path
+        >>> path="file://datasetinsights/../config.yaml" # relative local path
```

Comment:
Let's get rid of supports for relative local path. It's very error prone to use relative local path.